src/api/types.ts

```diff
@@ -148,6 +148,7 @@ export interface OHRIFormQuestionOptions {
   /**
    * maxLength and maxLength are used to validate text field length
    */
+  isSubmittable?: boolean;
```

Comment:
Since by default a field is submittable, I would change the flag to isNotSubmittable so that at the JSON level, users only pass the flag to fields that they don't want to be submitted.